### DISCUSSION TITLE
Flicker-Free Dimming keyboard shortcuts

### DISCUSSION THREAD

**Discussion by anventia** [seerge/g-helper#2402] [Ideas]
Hello, would it be possible to have an option to rebind Flicker-Free Dimming on 2024 Zephyrus models from (CTRL+FN+F7/8) to replace the normal Windows brightness control F7/8 (while still working with FnLock)? 

Or is this not possible since those are Windows shortcuts and not related to Asus so they can't be changed?

**Comment by seerge** [comment]
@anventia I didn't get it completely. Do you want ``FN + F7/F8`` control dimming instead ? Or just ``F7/F8`` (while FN-Lock is ON)

**Reply by anventia** [reply]
Fn+F7/8 when FN-Lock is Off (just like how windows brightness acts normally), and just F7/F8 when FN-Lock is On.

**Reply by MaoAnguita** [reply]
hello @seerge  I second this

I have a G16 (2024) and the keyboard brightness key controls "Windows brightness" and not "G-Helper Brightness"

<img width="880" alt="image" src="https://github.com/seerge/g-helper/assets/169324074/29b15fcf-1ebe-4083-909f-24b63038b1e4">

I hope with the image it makes sense.

I have all the asus services stopped and AC uninstalled 

<img width="166" alt="image" src="https://github.com/seerge/g-helper/assets/169324074/c01ae42b-2274-4c63-ac5f-af0ffbad5be2">

**Comment by phluxi** [comment]
I would also love to see that feature. I use flicker-free dimming exclusively so the hotkeys controling Windows default dimming makes them useless for me. Would be way better to have them as a shortcut to change flicker-free dimming without the need to open G-Helper UI.
